ghost/i18n/locales/context.json

```
@@ -170,7 +170,7 @@
     "Report this comment?": "Title of the modal to report a comment",
     "Retry": "When something goes wrong, this link allows people to re-attempt the same action",
     "Save": "A button to save",
-    "Search posts, tags and authors": "",
+    "Search posts, tags and authors": "Gönderileri, etiketleri ve yazarları arayın",
```

translations don't belong in context.json. Please revert this change
```suggestion
    "Search posts, tags and authors": "",
```